.travis/install.sh

```diff
@@ -10,9 +10,9 @@ fi
 
 if python --version 2>&1 | grep -q PyPy; then
   # cryptography requires PyPy >= 2.6, Travis CI uses 2.5.0
-  $SCAPY_SUDO pip install $PIP_INSTALL_FLAGS ecdsa mock
+  $SCAPY_SUDO pip install $PIP_INSTALL_FLAGS mock
 else
-  $SCAPY_SUDO pip install $PIP_INSTALL_FLAGS cryptography ecdsa mock
+  $SCAPY_SUDO pip install $PIP_INSTALL_FLAGS cryptography mock
```

Comment:
The version of Pypy used in Travis is not recent enough for cryptography. Please remove that change.